Makefile

```diff
@@ -54,4 +51,4 @@ run-test-rollback:
 wait-for-docker-postgres:
 	./wait_for_docker_postgres.sh
 
-test-patches: pull-latest-dev-postgres dev-postgres-pre-down dev-postgres-up wait-for-docker-postgres run-test-patches run-test-rollback dev-postgres-down
+test-patches: build-ddl-docker dev-postgres-down dev-postgres-up wait-for-docker-postgres run-test-patches run-test-rollback dev-postgres-down
```

Comment:
This was intentional, the test patches target is supposed to run through any patch scripts that are new, so it must pull the current latest main build rather than build its own because if it builds its own, the changes will all be ground-zeroed in and the patches won't be run.